src/orchestrator-tracing/src/lib.rs

```diff
@@ -251,8 +252,8 @@ impl NamespacedOrchestrator for NamespacedTracingOrchestrator {
                 tokio_console_retention,
             } = &self.tracing_args;
             args.push(format!("--log-filter={log_filter}"));
-            if *log_include_service_name {
-                args.push("--log-include-service-name".into());
+            if log_prefix.is_some() {
+                args.push(format!("--log-prefix={}d-{}", self.namespace, id));
```

Comment:
@benesch, this `d` is unfortunate, we can either:
- change the `compute` and `storage` namespace names
- just leave it `storage-` and `compute-`
- or take a `service_name` parameter in the `Orchestrator::namespace` method

Reply:
Oh let’s just leave it without the `d`! Naming these after the services exactly sounds like the right call!